{ 'cebit_greet_right': { 'head': [ { 'duration': 1.96,
                                     'name': u'cebit_slight_right',
                                     'positions': [-0.377, -0.0],
                                     'starttime': 2.013},
                                   { 'duration': 2.024,
                                     'name': u'ready_state',
                                     'positions': [-0.0, 0.0022],
                                     'starttime': 9.559}],
                         'left_arm': [],
                         'left_leg': [ { 'duration': 2.024,
                                         'name': u'cebit_upright',
                                         'positions': [ 0.0199,
                                                        -0.0901,
                                                        -0.6282,
                                                        1.2101,
                                                        0.6201,
                                                        -0.0899],
                                         'starttime': 0.0},
                                       { 'duration': 1.704,
                                         'name': u'ready_state',
                                         'positions': [ 0.0113,
                                                        -0.0746,
                                                        -0.7899,
                                                        1.1893,
                                                        0.6074,
                                                        -0.0704],
                                         'starttime': 11.699}],
                         'right_arm': [ { 'duration': 2.408,
                                          'name': u'cebit_greet_open',
                                          'positions': [ 1.1606,
                                                         1.3987,
                                                         1.0485,
                                                         -0.0689,
                                                         0.5998,
                                                         0.2598,
                                                         -1.56],
                                          'starttime': 2.004},
                                        { 'duration': 1.576,
                                          'name': u'cebit_greet_closed',
                                          'positions': [ 2.2849,
                                                         0.9283,
                                                         1.3037,
                                                         -0.1707,
                                                         0.6004,
                                                         0.2592,
                                                         -1.56],
                                          'starttime': 4.412},
                                        { 'duration': 1.5819999999999999,
                                          'name': u'cebit_greet_open',
                                          'positions': [ 1.1606,
                                                         1.3987,
                                                         1.0485,
                                                         -0.0689,
                                                         0.5998,
                                                         0.2598,
                                                         -1.56],
                                          'starttime': 5.988},
                                        { 'duration': 1.431,
                                          'name': u'cebit_greet_closed',
                                          'positions': [ 2.2849,
                                                         0.9283,
                                                         1.3037,
                                                         -0.1707,
                                                         0.6004,
                                                         0.2592,
                                                         -1.56],
                                          'starttime': 7.57},
                                        { 'duration': 1.954,
                                          'name': u'ready_state',
                                          'positions': [ 1.5135,
                                                         -0.7532,
                                                         0.2755,
                                                         -0.0212,
                                                         -0.0208,
                                                         0.2499,
                                                         -0.8168],
                                          'starttime': 9.543}],
                         'right_leg': [ { 'duration': 2.024,
                                          'name': u'cebit_upright',
                                          'positions': [ -0.0199,
                                                         0.0901,
                                                         0.6282,
                                                         -1.2101,
                                                         -0.6201,
                                                         0.0899],
                                          'starttime': 0.0},
                                        { 'duration': 1.704,
                                          'name': u'ready_state',
                                          'positions': [ -0.0113,
                                                         0.0746,
                                                         0.7899,
                                                         -1.1893,
                                                         -0.6074,
                                                         0.0704],
                                          'starttime': 11.683}],
                         'torso': [ { 'duration': 2.024,
                                      'name': u'cebit_right',
                                      'positions': [-0.1569, 0.0103],
                                      'starttime': 1.997},
                                    { 'duration': 2.024,
                                      'name': u'ready_state',
                                      'positions': [0.0001, -0.0002],
                                      'starttime': 9.606}]}}
